Flaky test: celu vjp

```
FAILED thunder/tests/test_grad.py::test_vjp_correctness_celu_torch_cuda_thunder.dtypes.float64 - AssertionError: Scalars are not close!

Expected 10.93902331508054 but got 10.939022012095851.
Absolute difference: 1.3029846890333374e-06 (up to 1e-07 allowed)
Relative difference: 1.1911343924434666e-07 (up to 1e-07 allowed)
```

cc @borda

--- a/thunder/tests/opinfos.py
+++ b/thunder/tests/opinfos.py
@@ -1649,6 +1649,7 @@ celu_opinfo = OpInfo(
     dtypes=(datatypes.floating,),
     sample_input_generator=get_elementwise_unary_with_alpha_generator(),
     torch_reference=_elementwise_unary_torch(torch.celu),
+    singularity_fn=lambda x: x,
     test_directives=(),
 )
 elementwise_unary_ops.append(celu_opinfo)

